lib/puppet/resource/type.rb

```diff
@@ -360,9 +360,6 @@ def set_resource_parameters(resource, scope)
     resource.add_parameters_from_consume
     inject_external_parameters(resource, scope)
 
-    resource_hash = {}
```

Comment:
Not sure what you are suggesting here. Should we update all method comments to state whether or not they are executed on the client side? If so, then someone else needs to do that (perhaps in scope for another JIRA). I don't know how to figure that out.